<ui version="4.0" >
 <class>SimulationGuiElement</class>
 <widget class="QWidget" name="SimulationGuiElement" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QWidget" native="1" name="modelControlWidget" >
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QPushButton" name="pbnStartModel" >
        <property name="text" >
         <string>Start Simulation...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QPushButton" name="pbnRemoveModel" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="lblRun" >
        <property name="text" >
         <string>Run Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="lblBatch" >
        <property name="toolTip" >
         <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">If an indicator batch (defined in the Results Manager)&lt;br>is selected, it will be executed over every &lt;br>year of the simulation being configured. &lt;br>It will executed after the simulation is completed.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Indicator Batch:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLineEdit" name="leRunName" />
      </item>
      <item row="1" column="2" >
       <widget class="QComboBox" name="cboOptionalIndicatorBatch" />
      </item>
     </layout>
     <zorder>pbnStartModel</zorder>
     <zorder>pbnRemoveModel</zorder>
     <zorder>lblRun</zorder>
     <zorder>lblBatch</zorder>
     <zorder>leRunName</zorder>
     <zorder>cboOptionalIndicatorBatch</zorder>
     <zorder>tabWidget</zorder>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="toolTip" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This scenario run will appear &lt;br>in the Results Manager under the name &lt;br>specified here. If not specified,&lt;br>the name will default to a combination &lt;br>of run id and date of run.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="simprogressWidget" >
      <attribute name="title" >
       <string>Simulation Progress</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Total progress:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="runProgressBarTotal" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="value" >
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" >
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>28</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="summaryCurrentYearValue" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cbYear" >
         <property name="text" >
          <string>Show progress within current year</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_year_progress" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string/>
         </property>
         <property name="flat" >
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" >
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Year progress:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="runProgressBarYear" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="value" >
               <number>24</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" >
            <item>
             <spacer name="horizontalSpacer" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>28</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_5" >
              <property name="text" >
               <string>Status:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="summaryCurrentModelValue" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="cbModel" >
            <property name="text" >
             <string>Show progress within current model</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gb_model_progress" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title" >
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3" >
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2" >
               <item>
                <widget class="QLabel" name="label_3" >
                 <property name="text" >
                  <string>Model progress:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QProgressBar" name="runProgressBarModel" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="value" >
                  <number>24</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6" >
               <item>
                <spacer name="horizontalSpacer" >
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType" >
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>28</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_6" >
                 <property name="text" >
                  <string>Status:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="summaryCurrentPieceValue" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text" >
                  <string>-</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Log</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>3</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>3</number>
       </property>
       <item row="0" column="0" >
        <widget class="QTextEdit" name="logText" >
         <property name="lineWidth" >
          <number>0</number>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="indicatorWidget" >
      <attribute name="title" >
       <string>Diagnostics</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" >
        <widget class="QSplitter" name="splitter" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QPushButton" name="diagnostic_go_button" >
          <property name="text" >
           <string>Diagnose...</string>
          </property>
         </widget>
         <widget class="QComboBox" name="diagnostic_indicator_type" >
          <item>
           <property name="text" >
            <string>map</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>table</string>
           </property>
          </item>
         </widget>
         <widget class="QComboBox" name="diagnostic_dataset_name" />
         <widget class="QComboBox" name="diagnostic_indicator_name" />
         <widget class="QComboBox" name="diagnostic_year" />
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QTabWidget" name="indicatorResultsTab" >
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_4" >
          <attribute name="title" >
           <string>Tab 1</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
